README.md

```diff
@@ -60,7 +60,7 @@ $T(n) = 13^i * T(n/13^i)$ + $$\left( \sum_{j=0}^{i-1} 13^j * 5 \right)$$ <br>
 To achieve the base case of 1, i needs to be equal to $log_{13}(n)$ <br>
 
 Now we have:  <br>
-$T(n) = 13^{log_{13}(n)} * T(n/13^{log_{13}(n)})$ + $$\left( \sum_{j=0}^{log_{13}(n)} 13^j * 5 \right)$$ <br>
+$T(n) = 13^{log_{13}(n)} * T(n/13^{log_{13}(n)})$ + $$\left( \sum_{j=0}^{log_{13}(n) - 1} 13^j * 5 \right)$$ <br>
 The sum can be reduced down to n/13 because the summation from 0 up until $log_{13}(n)$ -1, will be a constant number, so the summation would look like $13^0 * 5 + 13^1 * 5$ and so on until $13^{log_{13}(n) -1}$ is reached. The summation would then become some constant c plus $13^{log_{13}(n)} * 13^{-1}$ which equals n/13. <br>
```

Comment:
The summation is not equal to n/13. Asymptotically, your answer is true, but the answer is: n-1/12 not n/13. Please refer to "Geometric Series" to find the right answer.